.github/workflows/unitTests.yaml

```diff
@@ -91,10 +93,19 @@ jobs:
       NODE_OPTIONS: "--max_old_space_size=4096"
     steps:
     - name: Checkout code
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
     - name: Setup Environment
       uses: ./.github/actions/setupEnvironment
     - name: Execute Accepted Schema
       run: psql $PG_URL -f ./schema/accepted_schema.sql
     - name: Run Database Bootstrap
       run: yarn migrate up
+
+  uploadCoverageReports:
+    needs: [runUnitTests, runIntegrationTests]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Upload to Coveralls
+      uses: coverallsapp/github-action@v2
+      with:
+        parallel-finished: true
```

Comment:
The separate "save coverage report" steps for unit tests and integration tests upload the two separate coverage reports to coveralls. This final step runs at the end and tells coveralls that we're done uploading coverage reports for this run and it can start processing the results.

Reply:
Doh - I accidentaly deleted it in a rebase